README.md

```diff
@@ -1,7 +1,13 @@
-# Cluster Bootstrap
+# MLOps OpenShift Cluster Bootstrap
```

Comment:
I would just update it to `OpenShift Cluster Bootstrap`.  Nothing  in the repo is strictly MLOps related even though we do install some tools that are geared towards MLOps.